A1-2015/SystemAOO/Plumber.java

```diff
@@ -0,0 +1,50 @@
+import java.text.SimpleDateFormat;

```

Comment:
@amine-bellamkaddem @hangyao @kayvanhaverbeck The plumber file is used to build the pipe and filter network, I suggest starting the review from this file.